modules/globebrowsing/tile/chunktile.h

```diff
@@ -34,6 +34,12 @@ namespace openspace {
 namespace globebrowsing {
 
 struct ChunkTile {
+	ChunkTile() : tile(Tile::TileUnavailable) {};
```

Comment:
Seems like I need to have a default constructor for `ChunkTile` to be able to return it from `TileProvider::getChunkTile`